File: gluecodium/src/test/resources/smoke/defaults/output/android/com/example/smoke/InternalEnumDefaults.java
/*
 *
 */
package com.example.smoke;
import android.support.annotation.NonNull;
import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;
public final class InternalEnumDefaults {
    @NonNull
    public FooBarEnum publicField;
    @NonNull
    public List<FooBarEnum> publicListField;
    @NonNull
    FooBarEnum internalField;
    @NonNull
    List<FooBarEnum> internalListField;
    public InternalEnumDefaults() {
        this.publicField = FooBarEnum.FOO;
        this.publicListField = new ArrayList<>(Arrays.asList(FooBarEnum.FOO, FooBarEnum.BAR, FooBarEnum.BAZ));
        this.internalField = FooBarEnum.BAR;
        this.internalListField = new ArrayList<>(Arrays.asList(FooBarEnum.FOO, FooBarEnum.BAR, FooBarEnum.BAZ));
    }
}

File: gluecodium/src/test/resources/smoke/defaults_const/output/android-kotlin/com/example/smoke/InternalEnumDefaults.kt
/*

 *
 */

@file:JvmName("InternalEnumDefaultsExtensions")

package com.example.smoke


class InternalEnumDefaults {
    @JvmField var publicField: FooBarEnum
    @JvmField var publicListField: List<@JvmSuppressWildcards FooBarEnum>
    @JvmField internal var internalField: FooBarEnum
    @JvmField internal var internalListField: List<@JvmSuppressWildcards FooBarEnum>



    constructor() {
        this.publicField = FooBarEnum.FOO
        this.publicListField = mutableListOf(FooBarEnum.FOO, FooBarEnum.BAR, FooBarEnum.BAZ)
        this.internalField = FooBarEnum.BAR
        this.internalListField = mutableListOf(FooBarEnum.FOO, FooBarEnum.BAR, FooBarEnum.BAZ)
    }





}

